src/Symfony/Component/Security/Core/Authorization/AuthorizationChecker.php

```diff
@@ -80,7 +80,7 @@ final public function isGranted($attribute, $subject = null): bool
             // @deprecated since Symfony 5.4
             if ($this->alwaysAuthenticate || !$authenticated = $token->isAuthenticated(false)) {
                 if (!($authenticated ?? true)) {
-                    trigger_deprecation('symfony/core', '5.4', 'Returning false from "%s()" is deprecated, return null from "getUser()" instead.');
+                    trigger_deprecation('symfony/core', '5.4', 'Returning false from "%s::%s()" is deprecated, return null from "getUser()" instead.', \get_class($token), 'isAuthenticated');
```

Comment:
Let's prefer `get_debug_type()` if we want to output a class name.

```suggestion
                    trigger_deprecation('symfony/core', '5.4', 'Returning false from "%s::%s()" is deprecated, return null from "getUser()" instead.', get_debug_type($token), 'isAuthenticated');
```